tables/sw-ke-g1-2.ctb

```diff
@@ -0,0 +1,105 @@
+# liblouis: Swahili (Kenya) (Grade 1.2)
+#
+#-index-name: Swahili (Kenya) Grade 1.2
+#-display-name: Swahili (Kenya) Grade 1.2
+#
+#-author-name: Jake Kyle
+#-maintainer-name: Jake Kyle
+#-maintainer-email: jake@compassbraille.org
+#-updated: 2022
+#
+#+language:sw
+#+region:ke
```

Comment:
Should be "sw-KE" (see https://github.com/liblouis/liblouis/wiki/Table-discovery-based-on-table-metadata).

Also, just checking: this braille code only applies to Kenya?

Reply:
Done.

Yes Martin Kieti told me that there are differences in Swahili Braille code between countries where Swahili is spoken, so at present this table is only for Kenya.

Reply:
Cool.